### PR TITLE
Add possibility to force to use tmux with g:fzf_force_tmux

### PR DESCRIPTION
**Pull request by edi9999**

### CODE REVIEW

File: plugin/fzf.vim [hunk 1]
```diff
@@ -140,7 +140,8 @@ try
   else
     let prefix = ''
   endif
-  let tmux = !has('nvim') && s:tmux_enabled() && s:splittable(dict)
+  let forcetmux = get(g:, 'fzf_force_tmux', 0)
+  let tmux = (!has('nvim') && s:tmux_enabled() && s:splittable(dict)) || forcetmux
```
Comment (justinmk) [inline plugin/fzf.vim]: The option shouldn't "force". It should still check `s:tmux_enabled()` and fallback to `:term` if not running in tmux.

So a better name for the option might be `g:fzf_prefer_tmux`.

File: plugin/fzf.vim [hunk 2]
```diff
@@ -140,7 +140,8 @@ try
...
   let command = prefix.(tmux ? s:fzf_tmux(dict) : fzf_exec).' '.optstr.' > '.temps.result
 
   if has('nvim')
```
Comment (junegunn) [inline plugin/fzf.vim]: We're still using Neovim terminal even when it's not necessary.

**Comment by junegunn** [comment]
Thanks. To address the issues in the comments, I suggest the following:

```diff
diff --git a/plugin/fzf.vim b/plugin/fzf.vim
index 9361181..ba5fb25 100644
--- a/plugin/fzf.vim
+++ b/plugin/fzf.vim
@@ -140,10 +140,10 @@ try
   else
     let prefix = ''
   endif
-  let tmux = !has('nvim') && s:tmux_enabled() && s:splittable(dict)
+  let tmux = (!has('nvim') || get(g:, 'fzf_prefer_tmux', 0)) && s:tmux_enabled() && s:splittable(dict)
   let command = prefix.(tmux ? s:fzf_tmux(dict) : fzf_exec).' '.optstr.' > '.temps.result
 
-  if has('nvim')
+  if has('nvim') && !tmux
     return s:execute_term(dict, command, temps)
   endif
 ```

**Comment by edi9999** [comment]
Done in https://github.com/junegunn/fzf/pull/593

**Pull request closed without merging**